packages/editor/src/components/blocks/combobox/Combobox.tsx

```diff
@@ -57,7 +58,7 @@ const UiBlock = ({ label, value, visible, required, disabled, updateOnChange }:
   <div className='block-input'>
     <UiBlockHeader visible={visible} label={label} required={required} disabled={disabled} updateOnChange={updateOnChange} />
     <div className='block-input__input'>
-      <span>{value}</span>
+      <InputBadge value={value} badgeProps={badgeProps} className='badge-label' />
```

Comment:
you have anywhere this component right?
I mean `<InputBadge value={value} badgeProps={badgeProps} className='badge-label' />`
why not create a common badge component for canvas stuff (e.g. beside the UiBlock components), which predefined this badgeProps and classname?